### PR TITLE
Make mbedtls compile into `build/host`.

### PR DESCRIPTION
**Pull request by floitsch**
Also don't enable programs and tests for mbedtls.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -119,14 +119,20 @@ add_custom_target(
...
+
+enable_testing()
+add_subdirectory(tests)
+
 set(CMAKE_POLICY_DEFAULT_CMP0076 OLD)
+# There is no prefix, so this variable might be interfere with other libraries.
+# Eventually, we need to import mbedtls differently.
+set(ENABLE_TESTING CACHE BOOL OFF)
+set(ENABLE_PROGRAMS CACHE BOOL OFF)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}")
```
Comment (floitsch) [inline CMakeLists.txt:132]: Changing the output directory back is a bit brittle, if we try to add another subdirectory later.
I don't know of a much better way, though.

File: CMakeLists.txt [hunk 2]
```diff
@@ -119,14 +119,20 @@ add_custom_target(
   COMMAND echo ${TOIT_GIT_VERSION} > "${CMAKE_BINARY_DIR}/sdk/VERSION"
 )
 
+add_subdirectory(src)
+
+enable_testing()
+add_subdirectory(tests)
+
 set(CMAKE_POLICY_DEFAULT_CMP0076 OLD)
+# There is no prefix, so this variable might be interfere with other libraries.
```
Comment (kasperl) [inline CMakeLists.txt:128]: this variable -> ? (be more specific)

be interfere -> interfere

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: CMakeLists.txt (hunk 2 above)
Comment (floitsch) [reply CMakeLists.txt]: done. (reworded a bit).

**Pull request merged**